src/main/java/jenkins/plugins/git/GitSCMSource.java

```diff
@@ -75,14 +82,25 @@
 
     private final boolean ignoreOnPushNotifications;
 
+    private final GitRepositoryBrowser browser;
+
+    private final String gitTool;
+
+    private final List<GitSCMExtension> extensions;
+
     @DataBoundConstructor
-    public GitSCMSource(String id, String remote, String credentialsId, String includes, String excludes, boolean ignoreOnPushNotifications) {
+    public GitSCMSource(String id, String remote, String credentialsId, String includes, String excludes,
+            boolean ignoreOnPushNotifications, GitRepositoryBrowser browser, String gitTool,
```

Comment:
@KostyaSha Thanks. I'll fix to use @DataBoundSetters against two new args(browser, gitTool).